packages/react/src/components/headerNavigationMenu/HeaderNavigationMenu.module.scss

```diff
@@ -0,0 +1,36 @@
+.headerNavigationMenu {
+  display: flex;
+  width: 100%;
+  height: 64px;
+  background-color: var(--color-white);
+  border-bottom: 1px solid var(--color-black-20);
+}
+.headerNavigationMenuList {
+  width: 100%;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  gap: 0 var(--spacing-xs);
+  flex-wrap: wrap;
+}
+.headerNavigationMenuLinkContainer {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+.headerNavigationMenuLinkContent {
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
+.headerNavigationMenuLinkContentActive {
+  box-shadow: inset 0 -4px 0 0 var(--color-black-90);
+  .headerNavigationMenuLink {
+    font-weight: bold;
+  }
+}
+.headerNavigationMenuLink.headerNavigationMenuLink {
```

Comment:
My arch nemesis here again 🥹

Reply:
😆 I don't think we found a solution to do this differently?